vision/body_analysis/arcface/model/arcfaceresnet100-opset8-md5.txt

```diff
@@ -0,0 +1 @@
+MD5 (resnet100.onnx) = 66074b860f905295aab5a842be57f37d
```

Comment:
MD5 file name can remove "opset8"

Reply:
the md5 hash should be specific to the model file. each model will have a separate hash

Reply:
Correct, although we have been consolidating the md5 files as in this example: https://github.com/onnx/models/blob/master/text/machine_comprehension/bert-squad/model/bert-md5.txt